Jenkinsfile

```diff
@@ -174,7 +174,7 @@ def yottaBuildStep(target, compilerLabel) {
               setBuildStatus('PENDING', "build ${exampleName}", 'build starts')
               try {
                 // try to compile different configurations
-                execute("make CFLAGS=-DMBED_CONF_CMDLINE_USE_MINIMUM_SET=0")
+                execute("make CFLAGS=-DMBED_CONF_CMDLINE_USE_MINIMUM_SET=1")
```

Comment:
This seems unrelated. Intentional?

Reply:
well, not directly, but undirectly since it test that minimum set can be compiled without compiler warnings....

Reply:
note that this was originally typo - it should be verify minimum set but it did not...